ports/triggerrally/gameinfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <name>Trigger Rally</name>
```

Comment:
```suggestion
    <name>Trigger Rally</name>
    <path>Trigger Rally.sh</path>
```

Reply:
Upon second look, should this not have been:
```
<path>./Trigger Rally.sh</path>
```